### PR TITLE
Added presentation guidance to Week 1 Day 2 scheduled

### LINKED ISSUE #839: Add to the coursebook schedule guidance to a good presentation
Find some time in the schedule to discuss what makes good presentations, with more specific guidance on what makes a good research presentation and what makes a good project presentation

### PR DESCRIPTION
**Pull request by Amirk390**
Removed 15 mins from the - ***Research / TBC*** and added 15 mins long ***Presentation Guidance***

[Master referance - Week 1 - Curriculum Planning](https://github.com/foundersandcoders/master-reference/issues/822)

Fixed #839

### CODE REVIEW

File: coursebook/week-1/presentation-guidance.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+#### 1. Draw up a plan
++ Take a little time to think about the content. 
++ You’ll need to introduce yourself and your topic 
++ Make some key points and then sum it all up at the end.
+
+#### 2. Choose your tools
++ Try using visual examples.
+  + Mark down file
+  +  Exampl code
```
Comment (astroash) [inline coursebook/week-1/presentation-guidance.md]: Exampl -> Example

Comment (sophieeejw) [inline coursebook/week-1/presentation-guidance.md]: Example code?

File: coursebook/week-1/presentation-guidance.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+### Presenation Step-By-Step
+
+#### 1. Draw up a plan
++ Take a little time to think about the content. 
++ You’ll need to introduce yourself and your topic 
++ Make some key points and then sum it all up at the end.
+
+#### 2. Choose your tools
++ Try using visual examples.
+  + Mark down file
```
Comment (astroash) [inline coursebook/week-1/presentation-guidance.md]: Mark down -> Markdown

File: coursebook/week-1/presentation-guidance.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  +  Exampl code
+  +  Images
+
+#### 3. Design Tips
++ Once you’ve decided on the content, you’ll need to present it in a way that’s engaging for people to look at and easy on the eye.
```
Comment (astroash) [inline coursebook/week-1/presentation-guidance.md]: What's meant by "easy on the eye"? I'd recommend clarifying this point 
e.g.  visual/easy to comprehend/short paragraphs

File: coursebook/week-1/presentation-guidance.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    It’s just a way for you to **get used to speaking out loud and prevent yourself from becoming a waffling mess on the day.**
+
+
+#### 5. Calm your nerves
++ Even if you’re perfectly prepared and super organised, you’re still likely to get the jitters before you have to present.
++ Just remember that this is entirely normal, and the majority of people feel exactly the same way.
+
+#### 6. Clear and confident
++ A confident presenter doesn’t mumble or talk to the floor.
++ Even if you’re feeling anxious and nervy, lift your head up high and project your voice in a confident manner.
```
Comment (astroash) [inline coursebook/week-1/presentation-guidance.md]: nervy -> nervous

File: coursebook/week-1/presentation-guidance.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+You’ve made it to the end of the presentation. Well done! 
+
++ You can sum up your main points and bring your presentation to its natural end. 
++ This is your chance to show that you’ve thoroughly researched your topic and thought about your next steps.
+
+#### 8. Round it off
++ Factor in some extra time for a Q&amp;A session at the end.
++  Have a couple of questions up your sleeve that you can ask the audience to get a bit of interaction going, just in case you’re met with a wall of silence.
+
+#### 9. Last but not least
```
Comment (sophieeejw) [inline coursebook/week-1/presentation-guidance.md]: Also I would add source of reference if this is not written from scratch :)

File: coursebook/week-1/presentation-guidance.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
++ Just remember that this is entirely normal, and the majority of people feel exactly the same way.
+
+#### 6. Clear and confident
++ A confident presenter doesn’t mumble or talk to the floor.
++ Even if you’re feeling anxious and nervy, lift your head up high and project your voice in a confident manner.
++ Keep your pace steady and don’t rattle along at a lightning speed to get it all over more quickly.
++ Take time to breathe, and treat each slide as a way to have a brief pause. 
++ Having a glass of water to hand is also a good idea.
+
+#### 7. Sum it off
```
Comment (astroash) [inline coursebook/week-1/presentation-guidance.md]: Sum it off -> Sum it up/Conclusion/Summarise

**Comment by Amirk390** [comment]
Thanks @astroash @sophieeejw 
Fixed all the typos :smile:

File: coursebook/week-1/presentation-guidance.md [hunk 7]
```diff
@@ -9,12 +9,12 @@
...
 + Try using visual examples.
-  + Mark down file
-  +  Exampl code
+  + Markdown file
+  +  Example code
   +  Images
 
 #### 3. Design Tips
-+ Once you’ve decided on the content, you’ll need to present it in a way that’s engaging for people to look at and easy on the eye.
++ Once you’ve decided on the content, you’ll need to present it in a way that’s engaging for people to look at and easy to comprehend
```
Comment (astroash) [inline coursebook/week-1/presentation-guidance.md]: Missing full stop 🙈

Comment (Amirk390) [reply coursebook/week-1/presentation-guidance.md]: Woops, nice catch! @astroash

**Comment by Amirk390** [comment]
@astroash fixed :see_no_evil:

File: coursebook/week-1/presentation-guidance.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
+# What makes a good research presentation
+
+### Presenation Step-By-Step
+
+#### 1. Draw up a plan
++ Take a little time to think about the content. 
++ You’ll need to introduce yourself and your topic
```
Comment (astroash) [inline coursebook/week-1/presentation-guidance.md]: Missing fullstop...

File: coursebook/week-1/presentation-guidance.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+### Presenation Step-By-Step
+
+#### 1. Draw up a plan
++ Take a little time to think about the content. 
++ You’ll need to introduce yourself and your topic 
++ Make some key points and then sum it all up at the end.
+
+#### 2. Choose your tools
++ Try using visual examples.
```
Comment (astroash) [inline coursebook/week-1/presentation-guidance.md]: . -> :

Comment (Amirk390) [reply coursebook/week-1/presentation-guidance.md]: Am i missing something? :stuck_out_tongue_closed_eyes: 
what's wrong with this line?

**Comment by Amirk390** [comment]
@astroash Fixed 
hope is all good now :stuck_out_tongue_closed_eyes:

File: coursebook/week-1/presentation-guidance.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
+# What makes a good research presentation
+
+### Presenation Step-By-Step
```
Comment (claireinez) [inline coursebook/week-1/presentation-guidance.md]: Typo: presentation 😁

**Comment by Amirk390** [comment]
@claireinez Good catch :laughing: 
Fixed

File: coursebook/week-1/presentation-guidance.md (hunk 5 above)
Comment (shiryz) [reply coursebook/week-1/presentation-guidance.md]: I would add that as point 4

**Comment by Amirk390** [comment]
@astroash is that alright now? :smiley:

**Pull request merged**